### PR TITLE
Fix cents issues

### PR DESCRIPTION
**Pull request by ashkan18**
# Problem
When we are mapping Exchange's price related fields we used `amount` helper which would return formatted price strings BUT we kept calling them `itemTotalCents` and etc. which is pretty confusing.

# Solution
For each price field expose two fields from MP, one the actual `Int` cent value and another formatted one without `Cents` in their name. Basically now we have

```
itemsTotalCents
shippingTotalCents
taxTotalCents
commissionFeeCents
transactionFeeCents
buyerTotalCents
sellerTotalCents
itemsTotal
shippingTotal
taxTotal
commissionFee
transactionFee
buyerTotal
sellerTotal
```

cc: @sepans @jonallured this impacts your current setup when you show prices, you may need to switch to non `Cents` fields.

### CODE REVIEW

File: src/schema/ecommerce/types/order_line_item.js [hunk 1]
```diff
@@ -27,6 +27,10 @@ export const OrderLineItemType = new GraphQLObjectType({
       description: "Edition set on the artwork",
       resolve: ({ editionSetId }) => editionSetId,
     },
+    priceCents: {
+      type: GraphQLInt,
+      description: "Price in cents",
+    },
```
Comment (joeyAghion) [inline src/schema/ecommerce/types/order_line_item.js]: Unclear if this is the price of 1 item or the subtotal of `quantity` items. The description could clarify.

**Comment by orta** [comment]
Yeah, this is totally fine from a stitching perspective. We would merge in the original schema to get in the "XYZcents" then we would stitch in the resolvers we have here. 👍

**Comment by joeyAghion** [comment]
This works for me! I'm fearful that merging may break dependent apps that need to be demo-ed. If that's all clear, feel free to merge.

**Comment by orta** [comment]
This looks whole incremental, so shouldn't pose a problem

**Comment by ashkan18** [comment]
👍 merging!

**Comment by starsirius** [comment]
Nice! I always have this question of how much data presentation Metaphysics should be responsible for, but the price stuff seems to be pretty standard across consuming clients.

**Pull request merged**